notebook/.ipynb_checkpoints/00.terms.of.use.text.2.haml-checkpoint.ipynb

```diff
@@ -0,0 +1,612 @@
+      "\n",
+      "\n",
+      "\n",
+      "\n",
+      "            %p\n",
+      "              %span Last updated March 28, 2019.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# open a text file and read it line by line\n",
+    "source_file_path = '{}/{}'.format(inputDataFolder, inputFileName)\n",
+    "\n",
+    "\n",
+    "convert = False\n",
+    "\n",
+    "title = \"Adopt a Drain Grand River\"\n",
+    "subtitle = \"Website Terms of Use Agreement\"\n",
+    "t = False\n",
+    "st = False\n",
+    "print(line == title)\n",
+    "print('        %ol{:style => \"list-style:none;\"}')\n",
+    "with open(source_file_path,'r') as f:\n",
+    "    for line in f:\n",
+    "        line = line.strip()\n",
+    "    \n",
+    "        if line == title:\n",
+    "            convert = True\n",
+    "            \n",
+    "        if convert:    \n",
+    "            \n",
+    "            if isNormal(line) and t and st:\n",
+    "                # print('isNormal')\n",
+    "                print(formatNormal(line))\n",
+    "            \n",
+    "            if not t and isTitle(line, title):\n",
+    "                # print('isTitle')\n",
+    "                print(formatTitle(line, title))\n",
+    "                t = True\n",
+    "            \n",
+    "            if not st and isSubTitle(line, subtitle):\n",
+    "                # print('isSubTitle')\n",
+    "                print(formatSubTitle(line, subtitle))\n",
+    "                st = True\n",
+    "                \n",
+    "            if isIndex(line):\n",
+    "                # print('isIndex')\n",
+    "                print(formatIndex(line))\n",
+    "            \n",
+    "            if isSubIndex(line):\n",
+    "                # print('isSubIndex')\n",
+    "                print(formatSubIndex(line))\n",
+    "            \n",
+    "            if isSpace(line):\n",
+    "                #print('isSpace')\n",
+    "                print(formatSpace())\n",
+    "             \n",
+    "                \n",
+    "            if isBullet(line):\n",
+    "                # print('isBullet')\n",
+    "                print(formatBullet(line))\n",
+    "            \n",
+    "            \n",
+    "                \n",
+    "            \n",
+    "          \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.1"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
I don't think this file is meant to be tracked in Git.

Reply:
Could you provide a little more maybe you thoughts as to why?

Reply:
Here's the question being asked on Stack Overflow: https://stackoverflow.com/questions/36306017/should-ipynb-checkpoints-be-stored-in-git

It's also part of GitHub default ignore file for Python projects: https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
Yes, yes, yes! I  didnt know the check points are independent from the notebook. I will give it a go. T